docs/setup-cluster/deploy-cluster/index.rst

```diff
@@ -1,47 +1,64 @@
-###################
- Set Up Determined
-###################
+.. _installation-guide:
 
-To set up Determined, start by following the cluster deployment guide for your environment.
+###############################
+ Install and Set Up Determined
+###############################
+
+.. meta::
+   :description: Discover how to install and set up your Determined cluster locally, on AWS, on GCP, on Kubernetes, on Slurm or on premises.
+
+To install and set up Determined, follow the cluster deployment guide for your environment.
+
++--------------------------------------------------------+
+| Local                                                  |
++========================================================+
+| -  :doc:`basic`                                        |
```

Comment:
should we use semantic references on this page?

Reply:
done